File: FootScout-Vue.WebAPI/Models/DTOs/ClubAdvertisementCreateDTO.cs
using FootScout_Vue.WebAPI.Entities;

namespace FootScout_Vue.WebAPI.Models.DTOs
{
    public class ClubAdvertisementCreateDTO
    {
        public int PlayerPositionId { get; set; }
        public string ClubName { get; set; }
        public string League { get; set; }
        public string Region { get; set; }
        public SalaryRangeDTO SalaryRangeDTO { get; set; }
        public string ClubMemberId { get; set; }
    }
}

File: FootScout-Vue.WebClient/src/models/dtos/ClubAdvertisementCreateDTO.ts
import type { SalaryRangeCreateDTO } from "./SalaryRangeCreateDTO";

export interface ClubAdvertisementCreateDTO {
    playerPositionId: number;
    clubName: string;
    league: string;
    region: string;
    salaryRangeDTO: SalaryRangeCreateDTO;
    clubMemberId: string;
}